### DISCUSSION TITLE
Retrieving the "Test Details" information from Testflight

### DISCUSSION THREAD

**Discussion by dharnapahuja** [fastlane/fastlane#17897] [Q&A]
Hi,
I am using the API and calling `def get_builds(filter: {}, includes: "buildBetaDetail,betaBuildMetrics", limit: 10, sort: "uploadedDate", cursor: nil)`. But it complains that " A parameter has an invalid value - 'betaBuildMetrics' is not a valid relationship name (Spaceship::UnexpectedResponse)". What am I doing wrong?

My goal Is to be able to read the text in the "Test Details" sections for a build number on Testflight

**Comment by abardallis** [comment]
@dharnapahuja can you provide a little more code here? By the `def` it looks like you're defining your own `get_builds`? So something is either amiss with that, or there's not enough here to reproduce exactly what call you're making. 

For what it's worth, I'm able to get what I need for the 5 most recently uploaded builds to the betaGroup of interest with the snippet below (be sure to fill in your own `@beta_group_id` and `@app_id`).

```
builds = Spaceship::ConnectAPI.get_builds(filter: {
                                                    betaGroups: @beta_group_id,
                                                    app: @app_id,
                                                    expired: false,
                                                    processingState: 'VALID'
                                                  },
                                                  includes: 'buildBetaDetail,betaBuildMetrics,preReleaseVersion',
                                                  sort: '-uploadedDate',
                                                  limit: 5).body
```
